src/clj/tic_tac_toe_clojure/core.clj

```diff
@@ -4,58 +4,55 @@
 (declare start-game)
 (declare scored-moves)
 (declare score-move)
+(declare get-rows)
 
 (defn -main [& args]
   (println "Starting game")
   (start-game))
 
-(defn create-board []
-  (into [] (repeat 9 "_")))
+(defn get-square [number]
+  (* number number))
 
-(defn create-4x4-board []
-  (into [] (repeat 16 "_")))
+(defn get-square-root [board]
+  (int (Math/sqrt (count board))))
 
-(defn create-sized-board [size]
-  (into [] (repeat (* size size) "_")))
+(defn create-board [size]
+  (into [] (repeat (get-square size) "_")))
 
-(def numbered-board
-  ["0" "1" "2" "3" "4" "5" "6" "7" "8"])
+(defn numbered-board [size]
+  (for [cell (take (get-square size) (range))] (str cell)))
```

Comment:
Could probably simply this in a few ways:

1. `(take n (range))` -> `(range n)`
2. `(for [cell ...] (str cell))` -> `(map str ...)`

```clojure
(map str (range (get-square size)))
```